core-web/apps/dotcms-ui/src/app/portlets/dot-edit-page/components/dot-palette/store/dot-palette.store.ts

```diff
@@ -28,60 +28,31 @@ export interface DotPaletteState {
 
 @Injectable()
 export class DotPaletteStore extends ComponentStore<DotPaletteState> {
-    private isFormContentType: boolean;
-    private itemsPerPage = 25;
-    private contentTypeVarName: string;
-    private initialContent: DotCMSContentType[];
-
     readonly vm$ = this.state$;
-
-    // UPDATERS
-    private readonly setContentlets = this.updater(
-        (state: DotPaletteState, data: DotCMSContentlet[] | DotCMSContentType[]) => {
-            return { ...state, contentlets: data };
-        }
-    );
-
-    private readonly setContentTypes = this.updater(
-        (state: DotPaletteState, data: DotCMSContentType[]) => {
-            return { ...state, contentTypes: data };
-        }
-    );
-
-    private readonly setTotalRecords = this.updater((state: DotPaletteState, data: number) => {
-        return { ...state, totalRecords: data };
-    });
-
     readonly setFilter = this.updater((state: DotPaletteState, data: string) => {
         return { ...state, filter: data };
     });
-
     readonly setLanguageId = this.updater((state: DotPaletteState, data: string) => {
         return { ...state, languageId: data };
     });
-
     readonly setViewContentlet = this.updater((state: DotPaletteState, data: string) => {
         return { ...state, viewContentlet: data };
     });
```

Comment:
Why removing the lines to separate updaters?

Reply:
It could be something with the Eslint (or Webstorm). I don't remember doing that manually.